examples/load-gen.ts

```diff
@@ -383,17 +413,21 @@ const loadGeneratorOptions: BasicJavaScriptLoadGenOptions = {
    */
   loggerOptions: {
     /**
-     * Available log levels are TRACE, DEBUG, INFO, WARN, and ERROR.  DEBUG
+     * Available log levels are TRACE, DEBUG, INFO, WARN, and ERROR.  INFO
      * is a reasonable choice for this load generator program.
      */
-    level: LogLevel.DEBUG,
```

Comment:
Was there something you were trying to filter out by changing this to INFO?  My memory is rusty but I believe that I had set it to DEBUG because there were some retry messages that were useful to see in certain situations.

Reply:
Yes, I added some logs that are quite verbose to test the delay mechanic. There's no other debug logs AFAIK, unless they're coming from something outside the script.

Reply:
they would show up if a retry was triggered.  would it make sense to do your new logs at TRACE?